src/wovnio/wovnphp/SSI.php

```diff
@@ -9,18 +11,31 @@ public static function readFile($includePath, $rootDir=null) {
     }
 
     public static function readFileRecursive($includePath, $rootDir, &$limit) {
-      $ssi_include_regexp = '/<!--#include virtual="(.+?)"\s*-->/';
+      $ssiIncludeRegexp = '/<!--#include virtual="(.+?)"\s*-->/';
```

Comment:
as I see, we use both `$camelCase and `$this_var` conventions, should we use just only 1?

Reply:
Yes, tried to limit changes to minimum, so i used conventions that are used in each file. As I mentionned earlier I want to start cleaning up.

Thanks for comment.